docs/examples/frameworks/index.rst

```diff
@@ -0,0 +1,9 @@
+Sequence Processing
+===================
+
+.. toctree::
+   :maxdepth: 2
+
+   video/video_reader_simple_example.ipynb
```

Comment:
I would put all of this into video category. Not the frameworks.
Also I don't see any rst that gathers MXNet, TF and Pytorch examples.

Reply:
My bad, removed this file.